README.md

```diff
@@ -81,6 +81,19 @@ BELOW HERE BE DRAGONS
 %%                {ArchRegex, TargetFile, Sources}
 %%                {TargetFile, Sources}
 %%
+%%                Note that if you want to use any of the rebar3 variables
+%%                below you must MUST use a ${}-style to get the expansion
+%%                to work. e.g. to expand REBAR_DEPS_DIR, do something like:
+%%
+%%                {port_specs, [{"priv/nif.so",
+%%                               ["c_src/nif.c",
+%%                                "${REBAR_DEPS_DIR}/foo/bar.c"]}]}.
+%%
+%%                This is a _very_ good way to be able to use you code both
+%%                as a top level app and a dependency.
+%%
+%%                CAVEAT! Not using {} is broken for the moment.
```

Comment:
Does this imply a TODO for the future? If so, maybe add one? Also, should this addition be reflected in the README?